lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -443,6 +443,17 @@ function dosomething_reportback_view_entity($entity) {
     $subtitle = ($is_owner) ? $copy_vars['owners_rb_subtitle'] . " " . $incentive : $node->call_to_action;
     $link = '/' . drupal_get_path_alias('node/' . $node->nid);
 
+    // Set twitter text.
+    $twitter_text = token_replace(variable_get('dosomething_campaign_permalink_confirmation_owners_social_network_copy'), array('node' => $node));
```

Comment:
Just curious -- is this `owners_social_network_copy` variable only used as Twitter message ?